### PR TITLE
ignore dev-targeted crates in coverage report

### PR DESCRIPTION
**Pull request by AdamGS**

### CODE REVIEW

**Comment by gatesn** [comment]
also add ~/home and ./fuzz

**Comment by lwwmanning** [comment]
and should sort out pyvortex -- it has >0% coverage, but in python, so they don't run with `cargo test`

**Comment by AdamGS** [comment]
@lwwmanning mind if I punt pyvortex to a followup PR?

**Comment by lwwmanning** [comment]
> @lwwmanning mind if I punt pyvortex to a followup PR?

Not at all, punt away

**Review by lwwmanning — APPROVED** [review]

File: .github/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ignore-non-existing: true
+
+excl-line: "unreachable!"
+ignore:
+  - "bench-vortex/*"
+  - "fuzz/*"
+  - "home/*"
```
Comment (a10y) [inline .github/coverage.yml:7]: what the heck is `home` ?

Comment (gatesn) [reply .github/coverage.yml:7]: I think should be `/home` for whatever reason it holds the Cargo cache and gets included

Comment (AdamGS) [reply .github/coverage.yml:7]: seems like ignoring /* or ../* actually cleans that up

**Comment by AdamGS** [comment]
[Final run](https://coveralls.io/jobs/157594417) looks good to me, did realize that we had some `vortex-serde` leftovers so I cleaned that up as well.

**Pull request merged**